Tasks/GithubRelease/operations/EditRelease.ts

```diff
@@ -0,0 +1,45 @@
+import tl = require("vsts-task-lib/task");
+import { WebRequest, sendRequest, WebResponse } from "./webClient";
+import * as Utility from "./Utility";
+import { GetReleaseByTag } from "./GetReleaseByTag";
+import util = require("util");
+
+export async function editRelease(): Promise<WebResponse> {
```

Comment:
Instead of creating one file for each operation on Release. I would suggest creating a single class for release and map methods to the operation.